modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -131,16 +130,18 @@ In the EU, the POM file is:
     <repository>
       <id>Repository</id>
       <name>Corporate Repository</name>
-      <url>https://maven.eu1.anypoint.mulesoft.com/api/v1/organizations/ORGANIZATION_ID/maven</url>
+      <url>https://maven.eu1.anypoint.mulesoft.com/api/v3/organizations/ORGANIZATION_ID/maven</url>
       <layout>default</layout>
     </repository>
   </distributionManagement>
   ...
 ----
 +
-*Note:* The <id> and <name> elements are arbitrary names you create to identify the repository and any other organizational identifiers. The <id> value must be the same in the pom.xml file as in the ~/.m2/settings.xml file. The <id> connects the pom.xml file with information to log into the organization's URL.
+*Note:* The `<id>` and `<name>` elements are arbitrary names you create to identify the repository and any other organizational identifiers. The `<id>` value must be the same in the pom.xml file as in the `~/.m2/settings.xml` file. The `<id>` connects the pom.xml file with information to log in to the organization's URL.
```

Comment:
Again, this is not the format that we use for a Note notation (which is something delimited by --- coding...you'll need to check with another writer).
However, this could easily be merged into the running narrative.
```suggestion
Note that the `<id>` and `<name>` elements are arbitrary names that you create to identify the repository and any other organizational identifiers. The `<id>` value must be the same in the `pom.xml` file as in the `~/.m2/settings.xml` file. The `<id>` value connects the `pom.xml` file with information to log in the organization's URL.
```